src/openseadragon.js

```diff
@@ -1337,6 +1340,7 @@ function OpenSeadragon( options ){
 
             // INITIAL FLIP STATE
             flipped:                    false,
+            overlayContentFlipped:      false,
```

Comment:
We should think about what the correct default should be. I know in your case having the flip compensation off is correct, but what is the average person going to want? Or more specifically, what is the expected default behavior?

I guess I'm thinking having the compensation on by default makes sense... "I want my overlay text readable" is a pretty uncontroversial statement. If someone needs them to be flipped, that's more of a specific interest.

What do you think?

Reply:
Yeah readable overlays by default.